yoyodyne/train.py

```diff
@@ -424,7 +433,12 @@ def main() -> None:
     train_set.index.write(index)
     util.log_info(f"Index: {index}")
     train_loader, dev_loader = get_loaders(
-        train_set, dev_set, args.arch, args.batch_size
+        train_set,
+        dev_set,
+        args.arch,
+        args.batch_size,
```

Comment:
I think it may be time to write `_get_loaders_from_args` or whatever, and have that call `get_loaders`.